docs/src/resources/troubleshooting.md

```diff
@@ -1,5 +1,101 @@
 # Troubleshooting
 
-This guide helps you troubleshoot common issues with Runwasi.
+This guide helps you troubleshoot common issues with runwasi shims.
 
-[TODO]
+## Container Fails to Start
+
+If your wasm container fails to start, you might see events like these:
+
+```
+Events:
+  Type     Reason                  Age               From               Message
+  ----     ------                  ----              ----               -------
+  Normal   Scheduled               18s               default-scheduler  Successfully assigned default/wasi-demo-75d6bb666c-479km to minikube
+  Warning  FailedCreatePodSandBox  4s (x2 over 18s)  kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = failed to get sandbox runtime: no runtime for "wasmtime" is configured
+```
+
+#### Investigation Steps:
+
+1. **Check runtime class configuration**: Ensure your `RuntimeClass` is correctly configured.
+   ```bash
+   kubectl get runtimeclass -o yaml
+   ```
+
+2. **Check containerd configuration**: Verify that the containerd configuration has a runtime configured for your Wasm shim (e.g., wasmtime).
+   ```bash
+   cat /etc/containerd/config.toml
+   
+   # Look for a section like:
+   [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.wasmtime]
+      runtime_type = "io.containerd.wasmtime.v1"
+   ```
+
+3. Make sure the containerd shim binary is installed properly.
```

Comment:
What does this mean?

Reply:
I should clarify that I meant the shim binary is in the PATH